feature-libs/product-configurator/rulebased/components/attribute/show-options/configurator-show-options.component.ts

```diff
@@ -30,5 +33,33 @@ export class ConfiguratorShowOptionsComponent {
       this.attributeComponentContext.group,
       this.attributeComponentContext.attribute
     );
+    this.focusFirstValue();
+  }
+
+  protected focusFirstValue(): void {
+    this.configuratorCommonsService
+      .isConfigurationLoading(this.attributeComponentContext.owner)
+      .pipe(
+        filter((isLoading) => isLoading),
+        take(1),
+        switchMap(() =>
+          this.configuratorCommonsService
+            .isConfigurationLoading(this.attributeComponentContext.owner)
+            .pipe(
+              filter((isLoading) => !isLoading),
+              take(1),
+              delay(0) //we need to consider the re-rendering of the page
+            )
+        )
+      )
```

Comment:
could be worth checking if we could simplify this, for example
```
this.configuratorCommonsService
   .isConfigurationLoading(this.attributeComponentContext.owner)
   .pipe(
      distinctUntilChanged(),
      skip(2), // first isLoading=false, second is Loading=true, third is loading=false
      take(1), 
      delay(0) // we need to consider the re-rendering of the page
)
```

Reply:
Done